src/plugins/home/public/application/components/home.js

```diff
@@ -173,9 +176,12 @@ export class Home extends Component {
                 : 'homData--expanded'
             }`}
           >
+            {/*
+            Disable the card to add sample data on the home page
             <EuiFlexItem>
               <AddData addBasePath={addBasePath} features={addDataFeatures} />
             </EuiFlexItem>
```

Comment:
**question**: Why was this disabled? Is this coming from the original source code or we did some changes previously to this pull request?